Framework/Algorithms/test/ChangeBinOffsetTest.h

```diff
@@ -164,4 +166,43 @@ class ChangeBinOffsetTest : public CxxTest::TestSuite {
   std::string inputSpace;
 };
 
+class ChangeBinOffsetTestPerformance : public CxxTest::TestSuite {
+public:
+  // This pair of boilerplate methods prevent the suite being created statically
+  // This means the constructor isn't called when running other tests
+  static ChangeBinOffsetTestPerformance *createSuite() {
+    return new ChangeBinOffsetTestPerformance();
+  }
+  static void destroySuite(ChangeBinOffsetTestPerformance *suite) {
+    AnalysisDataService::Instance().clear();
+    delete suite;
+  }
+
+  ChangeBinOffsetTestPerformance() {
+    auto input = WorkspaceCreationHelper::Create2DWorkspaceBinned(10000, 1000);
+    AnalysisDataService::Instance().add("input", input);
+    auto inputEvent =
+        WorkspaceCreationHelper::CreateEventWorkspace(10000, 1000, 5000);
+    AnalysisDataService::Instance().add("inputEvent", inputEvent);
```

Comment:
@SimonHeybrock : just curious. I see that you intentionally put the workspaces in the ADS and then take the inputs from there. Have you observed any performance implications when doing that?